lib/bucket/index.ios.js

```diff
@@ -1,21 +1,24 @@
-import { NativeModules } from 'react-native'
+import { NativeModules } from 'react-native';
 
-const { RNSKBucket } = NativeModules
+const { RNSKBucket } = NativeModules;
 
 const Bucket = {
-  set: (key, value, groupName) => {
-    RNSKBucket.set(key, value, groupName)
+  setString: (key, value, groupName) => {
```

Comment:
How would you use Flow type ? Do you have any link to help me use it ?

I tried something like : 
```
  set: (key: String, value: (String | Object), groupName: String) => {
    console.log(' _______ type is ', typeof (value));
    if (typeof (value) === 'string') {
      RNSKBucket.setString(key, value, groupName);
    } else if (typeof (value) === 'object') {
      RNSKBucket.setDictionary(key, value, groupName);
    }
  },
```
But, in this way, you can give any type to value (that's why I added a else if)